client/containers/Pub/PubDocument/PubHistoricalNotice.tsx

```diff
@@ -25,12 +25,12 @@ type Props = {
 const PubHistoricalNotice = (props: Props) => {
 	const { pubData, historyData } = props;
 	const { communityData } = usePageContext();
-	const { releases, releaseNumber, isRelease } = pubData;
+	const { releases, releaseNumber, isRelease, isReview } = pubData;
```

Comment:
From `PubDocument` it looks like we skip showing this component when we have a review on our hands, so maybe we can skip these changes?